build/pom.xml

```diff
@@ -204,6 +201,25 @@
                         <useTabIndent>true</useTabIndent>
                     </configuration>
                 </plugin>
+                <plugin>
+                    <groupId>pl.project13.maven</groupId>
+                    <artifactId>git-commit-id-plugin</artifactId>
+                    <version>${git-commit-id-plugin.version}</version>
+                    <executions>
+                        <execution>
+                            <id>get-the-git-infos</id>
+                            <goals>
+                                <goal>revision</goal>
+                            </goals>
+                        </execution>
+                    </executions>
+                    <configuration>
+                        <verbose>true</verbose>
+                        <dateFormat>yyyy-MM-dd'T'HH:mm:ssZ</dateFormat>
+                        <generateGitPropertiesFile>true</generateGitPropertiesFile>
+                        <generateGitPropertiesFilename>${project.build.outputDirectory}/seata-git.properties</generateGitPropertiesFilename>
```

Comment:
includeOnlyProperties
^git.dirty$
^git.commit.time$
^git.build.version$
^git.build.time$
^git.branch$
^git.commit.id$

Reply:
Let me see if it's possible to configure it.

Reply:
Done, PTAL